Imagine/Filter/Loader/ScaleFilterLoader.php

```diff
@@ -13,9 +13,24 @@
  */
 class ScaleFilterLoader implements LoaderInterface
 {
-    public function __construct($dimentionKey = 'dim', $ratioKey = 'to', $absoluteRatio = true)
+    /**
+     * @var string
+     */
+    protected $dimensionKey;
+
+    /**
+     * @var string
+     */
+    protected $ratioKey;
+
+    /**
+     * @var bool
+     */
+    protected $absoluteRatio;
+
+    public function __construct($dimensionKey = 'dim', $ratioKey = 'to', $absoluteRatio = true)
     {
-        $this->dimentionKey = $dimentionKey;
```

Comment:
Oh, it still does, I plan on restructuring this at a later date - The ScaleFilter is inherited by the DownscaleFilter and UpscaleFilter both of which use slightly different parameters.